app/components/select-list.js

```diff
@@ -0,0 +1,84 @@
+import Ember from "ember";
+import ValidatableInput from 'ember-cli-html5-validation/mixins/validatable-input';
+
+export default Ember.Component.extend(ValidatableInput, {
+  content: null,
+  selectedValue: null,
+  pkg: null,
+  index: null,
+  flag: true,
+
+  didRender() {
+    this._super(...arguments);
+    if ( this.pkg != null && this.pkg.packageType != null && this.flag === true){
```

Comment:
In Ember we use convention like **this.get('VariableName')**... Use of **this.variableName** is not preferred.